server/notification-providers/ceredsms.js

```diff
@@ -0,0 +1,45 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class CeredSMS extends NotificationProvider {
+
+    name = "ceredsms";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
```

Comment:
nit, was changed for all others

```suggestion
        const okMsg = "Sent Successfully.";
```